### PR TITLE
Fix uglify error with pump

### PR DESCRIPTION
**Pull request by simoneduca**
This fixes[ this issue](https://github.com/terinjokes/gulp-uglify/issues/249#issuecomment-262406528). 

Please test `pump` isn't actually swallowing errors here preview.zooniverse.org/shakespearesworld/#!/

### CODE REVIEW

**Comment by eatyourgreens** [comment]
I'm going to remove myself from this because I don't know Gulp. Sorry.

**Comment by camallen** [comment]
Pretty sure we don't fix the issue you have linked. Its related sure but it's not our issue to fix.

**Comment by eatyourgreens** [comment]
Looks ok to me. So `pump` takes an array of gulp tasks and an error handler callback as args? If the previous build was throwing an error, what happened to that error here? I don't see any change in the list of gulp tasks that are actually being run, so I'd naively assume that whatever was broken is still broken, just not being reported any more.

File: gulp/tasks/browserify.js [hunk 1]
```diff
@@ -59,18 +60,23 @@ function buildScript(file) {
...
-            .pipe(gulpif(createSourcemap, sourcemaps.init()))
-            .pipe(gulpif(global.isProd, streamify(
-                uglify({
-                    compress: { drop_console: true }
-                })
-            )))
-            .pipe(gulpif(createSourcemap, sourcemaps.write('./')))
-            .pipe(gulp.dest(config.scripts.dest))
-            .pipe(gulpif(browserSync.active, browserSync.reload({ stream: true, once: true })));
+        return stream.on('error', function() {
```
Comment (eatyourgreens) [inline gulp/tasks/browserify.js]: Should this be `function(handleErrors)`?

Comment (eatyourgreens) [reply gulp/tasks/browserify.js]: That occurred to me after reading the example at https://github.com/terinjokes/gulp-uglify/tree/master/docs/why-use-pump

Comment (eatyourgreens) [reply gulp/tasks/browserify.js]: Out of interest, why has this anonymous callback replaced the error handler in the original code?

**Comment by eatyourgreens** [comment]
When I run `npm run build`, why don't I see this building a JS file in the `build/` directory?

**Comment by simoneduca** [comment]
@eatyourgreens you don't know gulp, but I don't think you need to for spotting something fishy: thanks! I'm having a look.

**Comment by simoneduca** [comment]
Superseded.

**Pull request closed without merging**